src/cmd/manage_nsfs.js

```diff
@@ -388,6 +388,10 @@ async function fetch_account_data(action, user_input) {
     // secret_key as SensitiveString
     data.access_keys[0].secret_key = _.isUndefined(data.access_keys[0].secret_key) ? undefined :
         new SensitiveString(String(data.access_keys[0].secret_key));
+    // allow to unset new_buckets_path
+    if (data.nsfs_account_config.new_buckets_path === '') {
+        data.nsfs_account_config.new_buckets_path = undefined;
+    }
```

Comment:
Just for the code style to match I would move lines 403-404 to this part (unset values) and add in the same way the unset.